src/platform/packages/shared/kbn-opentelemetry-attributes/src/resource_attributes/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export const RESOURCE_ATTR_SERVICE_NAME = 'service.name';
+export const RESOURCE_ATTR_SERVICE_NODE_NAME = 'service.node.name';
+export const RESOURCE_ATTR_SERVICE_ENVIRONMENT = 'service.environment';
```

Comment:
Do these conform to SemConv?

AFAIK, `service.environment` should be `deployment.environment.name`: https://opentelemetry.io/docs/specs/semconv/resource/deployment-environment/

Based on [the definition of ServiceNodeName](https://www.elastic.co/docs/reference/apm/agents/dotnet/config-core), should `service.node.name` be `container.name?